packages/clerk-js/src/ui/components/UserProfile/PhoneSection.tsx

```diff
@@ -114,6 +116,7 @@ const PhoneMenu = ({ phone }: { phone: PhoneNumberResource }) => {
   const card = useCardState();
   const { open } = useActionContext();
   const { user } = useUser();
+  const phoneId = phone.id;
```

Comment:
Just curious what the thought behind this assignment is

Reply:
A consistent identifier used below to ensure the actions below are scoped to the appropriate phone number.